scripts/build.sh

```diff
@@ -235,6 +235,16 @@ case "$product-$method-$platform" in
         -scheme "Firestore_IntegrationTests_$platform" \
         "${xcb_flags[@]}" \
         build
+
+    # Firestore_FuzzTests_iOS require recent Clang version in Xcode 9.
```

Comment:
This comment is missing an article and is unclear on how this is related to Xcode 9.

Maybe "Firestore_FuzzTests_iOS require a Clang that supports -fsanitize-coverage=trace-pc-guard"